src/DocumentationBrowserViewExtension/Docs/ArgumentNullException.html

```diff
@@ -0,0 +1,54 @@
+<!-- 
+    There are two versions of this HTML file. One that is part of the Dynamo Dictionary 
+    codebase and one that is part of the Dynamo codebase, used for displaying additional
+    error information in the Dynamo side panel.  
+    
+    These files need to kept in sync manually: if making edits to one version of the file,
+    make sure to update the other version as well.
+
+    Dynamo: https://github.com/DynamoDS/Dynamo
+    Dynamo Dictionary: https://github.com/DynamoDS/DynamoDictionary_React
```

Comment:
Hi @radumg, sorry to bother you. I was taking a look at finishing up this PR and had a question. About this comment, I checked the linked repo `DynamoDictionary_React` and could not find where the other version of this file is. Could you point me to where these can be found in that repo? Thanks

Reply:
@mmisol looks like they might still be here and not merged to dictioanry - @Amoursol - can you remind me what that final state was supposed to be? Did we decide to only have them in Dynamo:
https://github.com/horatiubota/DynamoDictionary_React/tree/dynamo_error_messages

Reply:
@mjkkirschner @mmisol - Final state was to have on both Dynamo Dictionary and inside DynamoBIM. Desire was to wrap the Dictionary into Dynamo itself further down the line.

Reply:
Also note we've had to ever-so-slightly tweak the `.html` docs in the DynamoCore repo because of using the IE browser, adding a header tag and some styling.